```
org.apache.calcite.test.LintTest > testLintLog() failure marker
FAILURE   0.1sec, org.apache.calcite.test.LintTest > testLintLog()
    java.lang.AssertionError: 
    Expected: an empty collection
         but: <[invalid git log message '[CALCITE-7577] Upgrade log4j to 2.25.4 to fix CVE-2025-68161'; contains 'fix' or 'fixes'; you should describe the problem, not what you did]>
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
        at org.apache.calcite.test.LintTest.testLintLog(LintTest.java:381)
```
You should remove 'fix' from your RP title. And JIRA title and commit message should be same as PR title.

--- a/gradle.properties
+++ b/gradle.properties
@@ -141,7 +141,7 @@ junit4.version=4.13.2
 junit5.version=5.9.1
 kafka-clients.version=2.1.1
 kerby.version=1.1.1
-log4j2.version=2.17.1
+log4j2.version=2.25.4
 mockito.version=3.12.4
 mongodb-driver-sync.version=4.10.2
 # 1.43.0 is the last version with Java 8 support

